content/java/concepts/calendar/terms/computeFields/computeFields.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.computeFields()'
+Description: 'Converts the current millisecond time value time to calendar field values in fields[].'
```

Comment:
This text is an exact copy of the oracle documentation. That is considered plagiarism. All docs content must be original. Please revise the content with your own words/definition and make any relevant changes to line 15, so that the explanations are in alignment.